Service/PlaylistFetcher.cs

```diff
@@ -153,7 +161,7 @@ Playlist GetPlaylistForPastDays(PlaylistProvider provider)
 
         Playlist LoadPlaylistFromCache(PlaylistProvider provider, DateTime date)
         {
-            if (provider.DontCache)
+            if (provider.AllowCaching)
```

Comment:
I think you forgot to negate it

Reply:
Yes I did :(
I will address this in the next update. Until then it still works but it will redownload them every time instead of taking them from cache